src/database/versions.ts

```diff
@@ -2,7 +2,7 @@ import moment from 'moment';
 
 import { SETTINGS_FILENAME } from '@config';
 import { LocalStorage } from '@types';
-import { noOp } from '@utils';
+import { noOp } from '@utils/noOp';
```

Comment:
Import from `@utils`

Reply:
Ill check this again, but it was causing a circular dependency issue with the `@utils/queries.ts` file